akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -60,36 +60,41 @@ class Http2Ext(private val config: Config)(implicit val system: ActorSystem) ext
 
     val effectivePort = if (port >= 0) port else 443
 
-    val serverLayer: BidiFlow[HttpResponse, ByteString, ByteString, HttpRequest, NotUsed] = {
-      val wrappedContext = WrappedSslContextSPI.wrapContext(httpsContext)
-      val tls = http.sslTlsStage(wrappedContext, TLSRole.server)
+    def http2Layer(): BidiFlow[HttpResponse, SslTlsOutbound, SslTlsInbound, HttpRequest, NotUsed] =
```

Comment:
We cannot reuse the flows any more because of the volatile var. With the akka-stream TLS change, we can get rid of the WrappedSslContext hack but we still have a problem the problem of the sidechannel between TLS and AlpnSwitch that we probably won't be able to solve cleanly.